torch/distributed/_composable/fsdp/_fsdp_init.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from torch._prims_common import DeviceLikeType
+
+
+def _normalize_device(device: DeviceLikeType) -> torch.device:
+    if isinstance(device, torch.device):
+        if device == torch.device("cuda"):
+            return torch.device("cuda", torch.cuda.current_device())
```

Comment:
looks like this still relies on cuda or `torch.cuda` to be present, to support device agnostic, should we add the `device_handle` logic to getattr from the `torch` namespace?

Reply:
Yes! I wanted to avoid the `device_handle` for now to move faster. I can only test CUDA, so I wanted to write everything first assuming CUDA. It should not be too hard to abstract it to `device_handle` in the future.